### PR TITLE
[CCI] Add new or remove extra tags and styles in `saved_objects_management` plugin

### PR DESCRIPTION
**Pull request by SergeyMyssak**
### Description

Added new or removed extra tags and styles in `saved_objects_management` plugin

### Issues Resolved

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3967

### Check List

- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ export class Relationships extends Component<RelationshipsProps, RelationshipsSt
         }
         color="danger"
       >
-        {error}
+        <p>{error}</p>
```
Comment (SergeyMyssak) [inline src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx:119]: `{error}` should be wrapped in `<p></p>`, in order to indicate a paragraph of text

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx [hunk 2]
```diff
@@ -918,16 +918,16 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
...
-          </span>
+          </>
         )}
         {indexPatternConflictsWarning && (
-          <span>
+          <>
             <EuiSpacer size="s" />
             {indexPatternConflictsWarning}
-          </span>
+          </>
```
Comment (SergeyMyssak) [inline src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx:930]: `<span></span>` tags are redundant and can be replaced with `<></>`

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.scss [hunk 3]
```diff
@@ -14,7 +14,3 @@
 .savedObjectsManagementImportSummary__errorCount {
   color: $euiColorDangerText;
 }
-
-.savedObjectsManagementImportSummary__icon {
-  margin-left: $euiSizeXS;
-}
```
Comment (SergeyMyssak) [inline src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.scss:20]: `.savedObjectsManagementImportSummary__icon` classname is not used anywhere and can be removed

File: src/plugins/saved_objects_management/public/management_section/object_view/components/intro.tsx [hunk 4]
```diff
@@ -44,12 +44,12 @@ export const Intro = () => {
       iconType="alert"
       color="warning"
     >
-      <div>
+      <p>
```
Comment (SergeyMyssak) [inline src/plugins/saved_objects_management/public/management_section/object_view/components/intro.tsx:47]: `<div></div>` should be replaced with `<p></p>`, in order to indicate a paragraph of text

File: src/plugins/saved_objects_management/public/management_section/object_view/components/not_found_errors.tsx [hunk 5]
```diff
@@ -76,13 +76,13 @@ export const NotFoundErrors = ({ type }: NotFoundErrors) => {
       iconType="alert"
       color="danger"
     >
-      <div>{getMessage()}</div>
-      <div>
+      <p>{getMessage()}</p>
+      <p>
```
Comment (SergeyMyssak) [inline src/plugins/saved_objects_management/public/management_section/object_view/components/not_found_errors.tsx:80]: `<div></div>` should be replaced with `<p></p>`, in order to indicate a paragraph of text

**Review by ashwin-pc — DISMISSED** [review]
Thanks for the change @SergeyMyssak This looks good :) can you resolve the changelog conflicts here?

**Review by joshuarrrr — COMMENTED** [review]
@SergeyMyssak Looks great! One minor question about where we may be able to simplify even further, to minimize `<></>`. BTW, in general, we prefer `<>` to `<Fragment>`.

**Comment by joshuarrrr** [comment]
I also resolved the changelog conflict.

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx (hunk 2 above)
Comment (ananzh) [reply src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx:930]: +1

Comment (SergeyMyssak) [reply src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx]: I reduced the number of `Fragment`, also I unintentionally overwrote @joshuarrrr merge commit, I apologize 😨

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx [hunk 6]
```diff
@@ -917,18 +917,10 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
...
-        {indexPatternConflictsWarning && (
-          <span>
-            <EuiSpacer size="s" />
-            {indexPatternConflictsWarning}
-          </span>
-        )}
+        {legacyFileWarning && <EuiSpacer size="s" />}
+        {legacyFileWarning}
+        {indexPatternConflictsWarning && <EuiSpacer size="s" />}
+        {indexPatternConflictsWarning}
```
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx:923]: @joshuarrrr, we can't write it that way, to write it that way we have to wrap the two components in `<></>`:
```jsx
{legacyFileWarning && (
  <>
    <EuiSpacer size="s" />
    {legacyFileWarning}
  </>
)}
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -21,6 +21,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Replace re2 with RegExp in timeline and add unit tests ([#3908](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3908))
 - Add category option within groups for context menus ([#4144](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4144))
 - [Saved Object Service] Add Repository Factory Provider ([#4149](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4149))
+- [Saved Objects Management] Add new or remove extra tags and styles ([#4069](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4069))
```
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply CHANGELOG.md]: It's done!

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (joshuarrrr) [reply src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx:923]: Yeah, I think I prefer that, although this whole thing could probably be refactored to be a bit cleaner.

**Review by joshuarrrr — DISMISSED** [review]

**Review by ananzh — APPROVED** [review]

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**